tableschema_sql/mapper.py

```diff
@@ -127,14 +127,20 @@ def convert_type(self, type):
             'geojson': None,
             'geopoint': None,
             'integer': sa.Integer,
-            'number': sa.Float,
+            'number': sa.Numeric,
             'object': None,
             'string': sa.Text,
             'time': sa.Time,
             'year': sa.Integer,
             'yearmonth': None,
         }
 
+        # Mysql dialect
+        if self.__dialect == 'mysql':
```

Comment:
@akariv 
I have no idea why but using `sa.Numeric` totally screw MySQL in tests. Like rounding and switching for 0/1 to False/True

Reply:
So maybe just for psql?

Reply:
It works with `sa.Numeric` for Postgres/SQLite and here I use fallback for MySQL